src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/Survey/SurveyURLProvider.swift

```diff
@@ -28,7 +28,7 @@ enum SurveyError: Error {
 
 	init(otpError: OTPError) {
 		switch otpError {
-		case .generalError, .invalidResponseError, .internalServerError, .otherServerError, .apiTokenExpired, .deviceTokenInvalid, .deviceTokenRedeemed, .deviceTokenSyntaxError:
+		case .generalError, .invalidResponseError, .internalServerError, .otherServerError, .apiTokenExpired, .deviceTokenInvalid, .deviceTokenRedeemed, .deviceTokenSyntaxError, .noNetworkConnection:
```

Comment:
So ppac, statistics, etc. will all get their own 'no network' errors as well?
This is a global error which should be handled on a higher level.

Reply:
I just asked UA about it, it make sense to update it through the app

Reply:
We already have the no network error, but almost every call is not handling it 😅.
But with that big refactoring ticket for the HTTPClient, we could face that, too.